### PR TITLE
fix(docs): warning about container image validation.

### PR DESCRIPTION
**Pull request by jvanz**
## Description

Updates the README file warning users about the potential issue when they enable this feature. The policy can be interrupted by the policy server if the evaluation took too much time.

Related to #93

### CODE REVIEW

**Review by viccuad — APPROVED** [review]

**Review by jhkrug — CHANGES_REQUESTED** [review]
Some suggestions.

File: README.md [hunk 1]
```diff
@@ -17,6 +17,16 @@ The policy has three settings:
 * `run_as_group`:  Controls which primary group ID the containers are run with. As well as the group in the securityContext from PodSpec.
 * `supplemental_groups`: Controls which group IDs containers add.
 * `validate_container_image_configuration`: A boolean value that allows the policy to validate the `USER` directive in the container image configuration. The default value is `false`.
+> [!WARNING]  
+> When users enable the container image validation, the policy will perform
+> host capability call to access the container image metadata stored in the
```
Comment (kkaempf) [inline README.md:22]:
```suggestion
> a host capability call to access the container image metadata stored in the
```

File: README.md [hunk 2]
```diff
@@ -17,6 +17,16 @@ The policy has three settings:
...
+> registry. Which involves network access. This can slow down the policy
```
Comment (kkaempf) [inline README.md:23]:
```suggestion
> registry. Which involves network access.
> This can slow down the policy
```

File: README.md [hunk 3]
```diff
@@ -17,6 +17,16 @@ The policy has three settings:
...
+> evaluation in a cold execution when this data is not store in the cache.
+> Therefore, the policy server can interrupt the evaluation if it takes too
+> long to complete. Pay attention to the `--policy-timeout` flag when starting
```
Comment (fabriziosestito) [inline README.md:26]: let's point the user to https://docs.kubewarden.io/reference/policy-evaluation-timeout#configuration

**Review by viccuad — CHANGES_REQUESTED** [review]
I would like to cut a new patch release with this README (including artifacthub-pkg.yml bmp)

**Review by viccuad — APPROVED** [review]

**Review by jhkrug — APPROVED** [review]
lgtm

**Pull request merged**